odl/util/utility.py

```diff
@@ -47,6 +47,7 @@
     'repr_string',
     'attribute_repr_string',
     'method_repr_string',
+    'nextpow2',
```

Comment:
Although tiny, I'd probably put this into `numerics.py`, if only to avoid littering this kitchen sink file.